schemas/documents/registry/buyModal.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  type: 'document',
+  name: 'buyModal',
+  title: 'Buy Modal',
+  __experimental_actions: ['update', 'create', /* 'delete', */ 'publish'],
```

Comment:
should we disable creation?
```suggestion
  __experimental_actions: ['update', /*'create', 'delete', */ 'publish'],
```